src/components/AnxietyTracker.jsx

```
@@ -202,6 +269,178 @@ const AnxietyTracker = ({ onBack }) => {
+            return (
+              <motion.path
+                key={part}
+                id={id}
+                d={pathData[part]}
+                fill={getFillColor(part)}
+                stroke={getStrokeColor(part)}
+                strokeWidth={getStrokeWidth(part)}
+                onClick={() => handleBodyPartClick(part)}
+                whileHover={{ scale: 1.03 }}
+                whileTap={{ scale: 0.98 }}
+                transition={{ type: "spring", stiffness: 300, damping: 15 }}
```

Biome (2.1.2)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)